api/envoy/api/v2/cluster/circuit_breaker.proto

```diff
@@ -46,6 +46,10 @@ message CircuitBreakers {
     // The maximum number of parallel retries that Envoy will allow to the
     // upstream cluster. If not specified, the default is 3.
     google.protobuf.UInt32Value max_retries = 5;
+
+    // The maximum number of connection pools per cluster that Envoy will concurrently support at
```

Comment:
Can you flesh this out with why someone might want to configure this setting with links to arch docs and/or other fields? I don't think it would be intuitive to most users why they would ever configure this.

Reply:
This might just be a link to what you added to circuit_breaking.rst below.